assets/js/pages/SettingsPage/SettingsPage.jsx

```diff
@@ -79,18 +104,8 @@ function SettingsPage() {
                 Get your key here 👇 and use it to register your first agents,
                 or to add new ones.
               </p>
-              <div className="w-full md:w-1/3 mb-8">
-                <Button
-                  type="primary"
-                  size="big"
-                  disabled={loading}
-                  onClick={() => setShowApiKey(true)}
-                >
-                  Get your key now!
-                </Button>
-              </div>
               <Transition
-                show={showApiKey}
+                show={Boolean(apiKey)}
```

Comment:
I think we can entirely remove this `Transition`.
It is not used

Reply:
The transition is used to gently display the api key when it's fetched, instead of displaying "directly" in the page, it's a nice visual touch